pkg/apis/policy/validation/validation.go

```diff
@@ -50,8 +50,20 @@ func ValidatePodDisruptionBudgetUpdate(pdb, oldPdb *policy.PodDisruptionBudget)
 func ValidatePodDisruptionBudgetSpec(spec policy.PodDisruptionBudgetSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	allErrs = append(allErrs, extensionsvalidation.ValidatePositiveIntOrPercent(spec.MinAvailable, fldPath.Child("minAvailable"))...)
-	allErrs = append(allErrs, extensionsvalidation.IsNotMoreThan100Percent(spec.MinAvailable, fldPath.Child("minAvailable"))...)
+	if spec.MinAvailable != nil && spec.MaxUnavailable != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath, spec, "MinAvailable and MaxUnavailable cannot be both set"))
+	}
+
```

Comment:
> The same behavior was possible before by setting minAvailable as being equal to spec.replicas

I haven't seen anywhere that sort of validation. Yeah, 100% is not disallowed. I am questioning whether it's the right approach though. What is the value of 100% if an admin will end up bypassing the PDB in order to drain a node?